### DISCUSSION TITLE
Firmware 20230820 possible bug

### DISCUSSION THREAD

**Discussion by doperst2** [UzixLS/zx-sizif-512#57] [General]
With this firmware, whenever i press nmi button, my AY goes crazy and emits noise until reset.
This happens after some warmup of Sizif (couple of minutes after a cold machine first boot).
Rolling back to previous firmware, the bug disappears completely.
For details, see zx-pk thread, please.

**Comment by tuFFo126** [comment]
I have to check what it's like in my case because I have this new firmware in one of the sizifs.  I used it for a while and didn't notice any problem, but it didn't work for a long time and you write about heating up.  I will check and write.

**Reply by tuFFo126** [reply]
I checked, I have no problems with this new firmware from August 20, 2023

**Comment by Mr-JimmyDeath** [comment]
I have not seen this issue with my D1 or a number of E1s that I have built.

Regards, James

**Comment by doperst2** [comment]
Never mind, i have swapped AY and the problem is gone.
